<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPlaybackTestClass</class>
 <widget class="QMainWindow" name="QPlaybackTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>回放测试</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="m_pWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1171</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnInit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>630</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnPlayback">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>630</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始回放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnStopAll">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>630</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止所有回放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnUninit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>630</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>反初始化</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="m_pEdtMsgTip">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>670</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>670</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>监控点编号：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>700</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>回放开始时间：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>730</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>回放结束时间：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>760</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>录像存储位置：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>790</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>传输协议：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>820</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>是否启用GPU硬解：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>secret：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>appkey：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>API网关端口：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>790</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化布局：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>820</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>抓图存储路径：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETCameraIndex">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>670</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCBLocation">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>760</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCBTransMode">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>790</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCBGpuMode">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>820</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETStartTime">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>700</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETEndTime">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>730</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCBLayout">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>788</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETPort">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>760</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETSnapDir">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>820</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETAppkey">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>670</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETSecret">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>700</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETIp">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>730</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>API网关IP地址：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>755</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QPlaybackTest.qrc"/>
 </resources>
 <connections/>
</ui>
